detection_rules/etc/version.lock.json

```diff
@@ -5624,32 +5638,32 @@
       }
     },
     "rule_name": "Suspicious LSASS Access via MalSecLogon",
-    "sha256": "67ef6987a1c26528568dffe812dec248441f5f35b4b4d04978fcf2f7499539ab",
+    "sha256": "c6dc6bde05788088400398e0436398ecb324245f53ddfc83e8ac6a01c1dc1a19",
     "type": "eql",
-    "version": 101
+    "version": 102
   },
   "7bcbb3ac-e533-41ad-a612-d6c3bf666aba": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Tampering of Bash Command-Line History",
-        "sha256": "66b54d6084fe3d3ddb4668acbfd676d08cc2735a6554121b7545833db254d29e",
+        "sha256": "2b96d18fa5abf049d0a09e9ae9d08ce9926fd025ff095f2a2ac87073602ec8d7",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.